appengine-jetty-managed-runtime/pom.xml

```diff
@@ -217,13 +217,24 @@
         <version>0.3.2</version>
         <executions>
           <execution>
-            <id>build</id>
+            <id>jetty-build</id>
             <phase>install</phase>
             <goals>
               <goal>build</goal>
             </goals>
             <configuration>
-              <imageName>appengine-mvn-opensource</imageName>
+              <imageName>gce-jetty:build</imageName>
```

Comment:
Because the Dockerfile cannot be parameterized, we create a temporary image labelled build based on the jetty.version.  The Dockerfile then has gce-jetty:build hard coded in it.